tests/src/Functional/TranslationTestBase.php

```diff
@@ -65,7 +65,6 @@ protected function setUp(): void {
     $role = $this->entityTypeManager->getStorage('user_role')->load('oe_translator');
     $permissions = $role->getPermissions();
     $permissions[] = 'administer menu';
-    $permissions[] = 'edit any page content';
```

Comment:
Why did you remove this?

Reply:
Because I previously added it as it was needed for `testRemoteTranslationRoute` but now that I switched to functional JS, I moved the permission in that base class. The permission itself is only temporary needed anyway as it's supposed to be replaced later.